index.html

```diff
@@ -1014,6 +1014,10 @@ <h2 id="BaseAudioContext">
               of the <a>AudioContext</a> is already <code>running</code>,
               resolve <em>promise</em>, return it, and abort these steps.
               </li>
+              <li>If the <a>AudioContext</a> is not <a>allowed to start</a>,
```

Comment:
I'm following the pattern used for the steps above: if the condition is true, stops the steps and return. That implies that otherwise, we should continue.